roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1070,6 +1070,26 @@ def set_container_facts_if_unset(facts):
 
     return facts
 
+def set_installed_variant_rpm_facts(facts):
+    """ Set RPM facts of installed variant
+        Args:
+            facts (dict): existing facts
+        Returns:
+            dict: the facts dict updated with installed_variant_rpms
+                          """
+    installed_rpms = []
+    for base_rpm in ['openshift', 'atomic-openshift', 'origin']:
```

Comment:
@detiber, originally this method was using the service_type however that proved to be a little complicated at upgrade time.  In the case of the 3.0->3.1 upgrade the service type can be different than what is actually installed.  If this looks OK I'd like to merge this in for QE to test.